odh/base/monitoring/overrides/prometheus-operator/overlays/servicemonitors/odh-components-servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  # labels:
+  #   component.opendatahub.io/name: prometheus
+  #   opendatahub.io/component: "true"
+  #   team: opendatahub
```

Comment:
actually... How is prometheus supposed to find this service monitor?

Reply:
sorry this is still WIP :(